### PR TITLE
Add new cable model and block state generation

### PR DESCRIPTION
**Pull request by kevinrpb**
Hi there!

I recently came across this mod and it's great! Big kudos!

However... Those cable blocks look funny to me (and others, #142). So here's a first approach at redesigning the model and textures to create real cables!

I'm more than anything looking for feedback on the models themselves, not so much the implementation (yet), but feel free to point out any issues or concerns 😊. I do have a couple things that need improvement (eg. caching shapes, updating state from network where we already have the 'connections'...).

Here's how the cables look:

Cheers

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Wed Jul 24 21:07:16 EDT 2024
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```
Comment (kevinrpb) [inline gradle/wrapper/gradle-wrapper.properties:3]: Mb I didn't realized I changed this. Will revert.

**Comment by TeamDman** [comment]
Preview looks cute! 
A pic of segments that don't touch an inv would be nice too

Can't review rn, but I probably want the dynamic cables to be a separate block with a simple conversion crafting recipe to let people have their choice of full block vs these

**Comment by kevinrpb** [comment]
> A pic of segments that don't touch an inv would be nice too

Ah, makes sense. Basically the 'connections' would only appear when the cable can connect to the neighbour. Otherwise only the 'core' shows up. Here it is:



> I probably want the dynamic cables to be a separate block with a simple conversion crafting recipe to let people have their choice of full block vs these

That is also sensible. Which one would you have be the default?

**Comment by kevinrpb** [comment]
Cable 'block' made it back:

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (kevinrpb) [reply gradle/wrapper/gradle-wrapper.properties:3]: fixed

**Review by TeamDman — APPROVED** [review]

**Pull request merged**